search-solr/src/main/java/edu/unc/lib/boxc/search/solr/responses/SearchResultResponse.java

```diff
@@ -73,6 +74,14 @@ public void setResultCount(long resultCount) {
         this.resultCount = resultCount;
     }
 
+    public String getMinimumSearchYear() {
```

Comment:
I feel like this should be minimumDateCreatedYear to make it clear what field it contains. Although i'm also sort of wondering if we'd want a more generic name and to make it into a map or something in case we need to present multiple date filters at some point. But that is probably overkill at this point, we could always rename is later